Account for the removal of asm/system.h. It is apparently not required here.
http://git.kernel.org/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=0195c002

--- mxser.c~	2012-07-19 18:06:20.131383739 +0200
+++ mxser.c	2012-07-19 18:08:22.275349975 +0200
@@ -40,7 +40,6 @@
 #include <linux/bitops.h>
 #include <linux/slab.h>
 
-#include <asm/system.h>
 #include <asm/io.h>
 #include <asm/irq.h>
 #include <asm/uaccess.h>
